### DISCUSSION TITLE
2x BC2500 8 (V1) an HMS-1600-4t / HMS-2000-4t

### DISCUSSION THREAD

**Discussion by mastertape** [noone2k/hm2500pub#11] [General]
hallo hat hier jemand Erfahrungswerte wie sich das system verhält wenn ich beide Akkus an einen Hoymiles HMS-4t anschließe?
Ich habe hier beide modelle zur Auswahl. Ich hatte ganz am Anfang mal einen Versuch getsartet wobei sich beide dinger dann resetet haben und im WR übelst klingende Meldungen kamen... Overload oder sowas... Es war ganz am Anfang... wo ich noch keine Erfahrungswerte hatte... Mittlerweile bin ich mir gar nicht mehr so sicher ob vielleicht damals vielleicht ein anderes problem vorhanden war wie eine Netz ober oder unterspannung was hier schonmal vorkommt...

Daher wäre ich für ein paar Rückmeldungen Erfahrungswerte andwrer Dankbar. Es gab dazu auch schon mal ein paar unterhaltungen im Gist... Hier ist die Frage dann was nachhaltiger ;)

**Comment by noone2k** [comment]
ungedrosselt ziehen die teile > 400W / port am speicher ... die speicher schalten ab ...
hatte das mit nem hms-1800 ... der hms-1600 sollte grenzwertig sein ... 

bin u.a. deswegen auf ein hm-1500 umgestiegen ...

**Reply by noone2k** [reply]
und da gibt es auch das masse problem ...
so kann es sein, das versucht wird die 2x400 watt auf einem port durchzureichen ...

ergebniss das gleiche ... wr zuviel input / kiste zuviel output .... beide gehen in den schutzmodus ( blinken der poweranzeige )

**Comment by eshopsrc** [comment]
Das Problem lässt sich nur mittels einschaltstrombegrenzer regeln … leider … hat mich einen WR gekostet

**Reply by mastertape** [reply]
was haltet ihr von der lösung hier?
https://solarmodule-gladbeck.de/produkt/einschaltstrombegrenzung-fuer-balkonkraftwerke-mit-speicher/

**Reply by mastertape** [reply]
Der Herbst nah und das mit den  Sonnenstunden ausreizen  wird komplexer.... Da ich eh schon alles da habe will ich mich langsam aber sicher ans ac laden rann machen....

Ich frage mal in die runde hat hier jemand erfahrungen mit WR´s  wo ich problemlos die V1 er akkus an einen WR hängen könnte?

Es muss ja kein hoymiles sein -was wäre mit nem anderen 1kw oder 2kw wr. Gibt es da vielleicht was alternatives (was steuerba ist)
Es gibt ja auch diese Lumentree wr´s

Sprich ich überlege das evtl anders anzugehen... wenn ich sicher wäre das an einen hm-1500er beide Dinger drann könnrten würde ich mir da einen och holen - ansonsten überlege ich nach einer alternative wo ich beide akkus über einen wr auf null steuern kann,,,
